testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SimpleSubsystemsTestCase.java

```diff
@@ -43,6 +44,10 @@
 public class SimpleSubsystemsTestCase extends ContainerResourceMgmtTestBase {
 
     @Test
+    @Ignore
+    /*
+     * The jaxrs subsystem is no long empty.
```

Comment:
'no long**er**' - important I know :)

Reply:
Oh noooo!!! I might have to push that off to WF 20!!

Reply:
Ok, I went the extra mile. ;-)